thrimshim/thrimshim/main.py

```diff
@@ -184,6 +184,7 @@ def get_row(ident):
 @authenticate
 def update_row(ident, editor=None):
 	new_row = flask.request.json
+	override_changes = 'override_changes' in new_row
```

Comment:
this will let us override changes if I set `override_changes: false`, which is surprising.
Let's check it's there and true.
```suggestion
	override_changes = new_row.get('override_changes', False)
```